Chummer/data/weapons.xml

```diff
@@ -14843,6 +14843,30 @@
       <hide />
       <spec>Machine Guns</spec>
     </weapon>
+    <weapon>
+      <id>4629315e-9d09-4d04-92ac-3bad46675522</id>
+      <name>Ontario Arms Sling-Shot</name>
+      <category>Underbarrel Weapons</category>
+      <type>Ranged</type>
+      <conceal>6</conceal>
+      <accuracy>6</accuracy>
+      <damage>As Ammo</damage>
+      <mode>SA</mode>
+      <ammo>10(m)</ammo>
```

Comment:
Are you sure this is correct, an internal magazine with a size of 10 for a slingshot?